server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -673,9 +672,10 @@ public void testGroupByUnknownResultColumn() throws Exception {
     @Test
     public void testGroupByUnknownGroupByColumn() throws Exception {
         this.setup.groupBySetup();
-        Assertions.assertThrows(Exception.class,
-                                () -> execute("select max(birthdate) from characters group by details_ignored['lol']",
-                                              "Cannot GROUP BY type: undefined"));
+        assertThrows(() -> execute("select max(age) from characters group by details_ignored['lol']"),
```

Comment:
I meant to change the query to have a cast. For the error case the analyzer test is good enough. But a integration test that confirms that it works with cast should be here additionally

Reply:
Sorry i missed that.